### DISCUSSION TITLE
Is it possible to do batchprocessing on anomalib models, especially patchcore?

### DISCUSSION THREAD

**Discussion by amal-r1** [open-edge-platform/anomalib#591] [Q&A] [answered]
If possible, could someone tell me how can it be done?

**Comment by ashwinvaidya17** [comment]
@amal-r1 can you clarify what do you mean by batch processing?

**Comment by amal-r1** [comment]
I would like to pass multiple images to the model at the same time and get inference from them simultaneously.

**Comment by ashwinvaidya17** [comment, marked as answer]
This might address your usecase https://github.com/openvinotoolkit/anomalib/blob/main/tools/inference/lightning_inference.py. You can change the batch size in the Dataloader (line 70).

**Reply by watertianyi** [reply]
I use /tools/inference/torch_inference.py for inference. How should I set up multi-image inference?
